### PR TITLE
WIT visual redesign part 2

### PR DESCRIPTION
**Pull request by jameswex**
* Motivation for features / changes

Continuing visual redesign ahead of WIT 1.0 launch.

* Technical description of changes

Removed old png assets (no longer used), and therefore removed hide-images mode
Removed tf-regression-table (put table directly in dashboard HTML to match styles with non-regression tables)
Moved controls around to match latest mocks

* Screenshots of UI changes

wit1

wit2

wit3

* Detailed steps to verify changes work correctly (as executed by you)

Run through all 4 demos to ensure functionality and correct look and feel.
Still need to redesign PD plots area, which will be its own smaller PR.
Still need to do final design tweaks (font sizes, spacing, and so on).

* Alternate designs / implementations considered

### CODE REVIEW

**Comment by jameswex** [comment]
@tolga-b please review

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 1]
```diff
@@ -2841,6 +3101,81 @@ <h2>Create a distance feature</h2>
...
+                  this.getBinaryClassificationCount(inferenceStats, a);
+            } else {
+              return this.getMulticlassClassificationCount(
+                inferenceStats, b) - this.getMulticlassClassificationCount(
+                  inferenceStats, a);
+            }
+          } else if (this.selectedFeatureSort == 'Alphabetical') {
+            return this.getPrintableValue_(a).localeCompare(this.getPrintableValue_(b));
+          } else if (this.selectedFeatureSort == 'Accuracy') {
+            console.log('a acc' + this.getAccuracyModelIndex(
```
Comment (tolga-b) [inline tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: If not intentional I would remove console.log.

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 2]
```diff
@@ -3806,7 +4157,10 @@ <h2>Create a distance feature</h2>
       /**
        * Handler for when running inference, calls the backend.
        */
-      inferClicked_: function() {
+      inferClicked_: function(event) {
+        if (event != null) {
```
[the remark this answers is not in the document]
Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: this isn't necessary anymore (added it when button was embedded in another button but removed that). removed this code

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html (hunk 1 above)
Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: removed

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 3]
```diff
@@ -2316,11 +2608,18 @@ <h2>Create a distance feature</h2>
         return stats.probSameValue;
       },
 
+      visModeChanged_: function(mode) {
+        mode == 'dive' ?
```
[the remark this answers is not in the document]
Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: acknowledged, we will tackle in the PDP PR

**Comment by tolga-b** [comment]
@nfelt This mostly looks good. I think you can start checking while I do some final tests.

**Comment by jameswex** [comment]
@tolga-b fixed confusion matrix label issue you saw

@nfelt ready for your review

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 4]
```diff
@@ -958,18 +1064,72 @@
...
+        text-transform: none;
+      }
+      .rotated-icon {
+        transform: rotate(270deg);
+      }
+      paper-toggle-button {
+        font-size: 14px;
+        --paper-toggle-button-checked-button-color: white;
+        --paper-toggle-button-label-color: #3C4043;
+        }
```
Comment (tolga-b) [inline tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: is this bracket extra?

Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: good catch. removed

**Pull request merged**